data-prepper-core/src/test/java/org/opensearch/dataprepper/peerforwarder/RemotePeerForwarderTest.java

```diff
@@ -78,7 +78,7 @@ void test_forwardRecords_with_two_local_ips_should_process_record_two_record_loc
     void test_forwardRecords_with_one_local_ip_and_one_remote_ip_should_process_record_one_record_locally() {
         AggregatedHttpResponse aggregatedHttpResponse = mock(AggregatedHttpResponse.class);
         when(aggregatedHttpResponse.status()).thenReturn(HttpStatus.OK);
-        when(peerForwarderClient.serializeRecordsAndSendHttpRequest(anyCollection(), anyString(), anyString())).thenReturn(aggregatedHttpResponse);
+        when(peerForwarderClient.serializeRecordsAndSendHttpRequest(anyCollection(), anyString(), anyString(), anyString())).thenReturn(aggregatedHttpResponse);
```

Comment:
These tests should verify the actual input strings.

Reply:
We can check if records exists in the test on line 64 as both records in the test are processed locally. In this test I'm not sure how we can test it as we are not sure of which record will be processed locally. 
Is there a hamcrest method for it?